### PR TITLE
replace find command in Jenkinsfile

### PR DESCRIPTION
**Pull request by ddsharpe**
OEL 8-slim has removed the command "find".  To support OEL 8-slim base images, this change replaces the "find" command used to remove *.class files from the WDT Jython run.

### CODE REVIEW

File: tests/src/test/java/com/oracle/weblogic/imagetool/tests/ITImagetool.java [hunk 1]
```diff
@@ -937,4 +937,49 @@ void updateImageWithServerJRE(TestInfo testInfo) throws Exception {
             assertFalse(imageId.isEmpty(), "Image was not created: " + tagName);
         }
     }
+
+    /**
+     * Create an image with WDT Model only on OEL 8-slim
```
Comment (rjeberhard) [inline tests/src/test/java/com/oracle/weblogic/imagetool/tests/ITImagetool.java:942]: Very minor point... They no longer call it "Oracle Enterprise Linux", it's just "Oracle Linux", so OL 8-slim.

**Review by rjeberhard — APPROVED** [review]

**Comment by rjeberhard** [comment]
Approving with the extremely minor comment that "oel" -> "ol".

**Review by mriccell — APPROVED** [review]

File: imagetool/src/main/resources/docker-files/Create_Image.mustache [hunk 2]
```diff
@@ -207,7 +207,7 @@ RUN cd {{{tempDir}}}/opatch \
         -domain_type {{domainType}} \
         {{{wdtVariableFileArgument}}} {{{wdtModelFileArgument}}} {{{wdtArchiveFileArgument}}} \
         && rm -rf {{{wdt_home}}}/weblogic-deploy/logs \
-        && find {{{wdt_home}}}/weblogic-deploy/lib/python -name "*.class" -exec rm  {} \;
+        && shopt -s globstar && rm {{{wdt_home}}}/weblogic-deploy/lib/python/**/*.class
```
[the remark this answers is not in the document]
Comment (ddsharpe) [reply imagetool/src/main/resources/docker-files/Create_Image.mustache]: thanks, fixed.

File: tests/src/test/java/com/oracle/weblogic/imagetool/tests/ITImagetool.java (hunk 1 above)
Comment (ddsharpe) [reply tests/src/test/java/com/oracle/weblogic/imagetool/tests/ITImagetool.java]: Thanks, fixed.

**Review by lfeigen — APPROVED** [review]

**Pull request merged**